src/amo/containers/DetailPage.js

```diff
@@ -1,37 +0,0 @@
-import React from 'react';
-import { compose } from 'redux';
-import { connect } from 'react-redux';
-
-import AddonDetail from 'amo/components/AddonDetail';
-import { UNKNOWN } from 'core/constants';
-import { loadAddonIfNeeded, safeAsyncConnect } from 'core/utils';
-
-
-export class DetailPageBase extends React.Component {
-  render() {
-    return (
-      <div className="DetailPage">
-        <AddonDetail {...this.props} />
-      </div>
-    );
-  }
-}
-
-export function mapStateToProps(state, ownProps) {
-  const { slug } = ownProps.params;
-  const addon = state.addons[slug];
-  const installation = state.installations[addon.guid] || { status: UNKNOWN };
-  return {
-    addon,
-    ...addon,
```

Comment:
I think there's actually an issue open for this (saying we should stop/avoid using `...addon` spread) so if that's true we should at least reference #1416... maybe something in disco is still using it though.